src/lib/features/scheduler/schedule-services.ts

```diff
@@ -131,16 +132,16 @@ export const scheduleServices = async (
     );
 
     schedulerService.schedule(
-        () => {
-            clientMetricsServiceV2.bulkAdd().catch(console.error);
+        async () => {
+            await clientMetricsServiceV2.bulkAdd().catch(console.error);
```

Comment:
`() => clientMetricsServiceV2.bulkAdd().catch(console.error);` is exactly the same since the last statement returns Promise

Reply:
but there's no return... I could add one, right?

Reply:
In general it's more idiomatic to do  `() => Promise.resolve()` over `async () => { await Promise.resolve(); }`

Reply:
() => Promise.resolve() returns the promise when you don't add {} around it

Reply:
`() => Promise.resolve()` is more idiomatic than `() => { return Promise.resolve() }`